### DISCUSSION TITLE
Introductions 👋

### DISCUSSION THREAD

**Discussion by phipsae** [BuidlGuidl/batch11.buidlguidl.com#1] [Announcements]
👋 Welcome Batch 11 Builders!

Philip here, I've been a proud member of BuidlGuidl for the past year now. I am developer, love to learn everything about smart contracts and support the batches program.

Hope that the batch will be a very fun project for you all to go through. If you have any questions, feel free to ask anytime!

**Comment by cypherpepe** [comment]
Hi, team! My name is Cypherpepe.
I recently completed SpeedRunEthereum and really enjoyed exploring Scaffold-ETH and Solidity.  
I’m looking forward to deepening my knowledge in dApp development and improving my GitHub collaboration skills.  
Excited to work together and share ideas.

**Reply by phipsae** [reply]
Welcome, glad to have you here!

**Comment by Asante-ntiro** [comment]
Hello Batch 11 Builders and Mentors! 👋 

My name's Asante, I just joined the BuidlGuidl. I like computers and can sometimes be referred to as a "developer"
I love, love, LOVE the Scaffold-ETH project and can't wait to make something meaningful out of this journey with you all!
Hoping to make great friends along the way!

**Reply by phipsae** [reply]
Nice to have you! Welcome :)

**Comment by FrankiePower** [comment]
Greetings to Fellow Batch 11 Builders and Mentors!

My name is Frankie Power, i am super excited to continue this journey with you all. i recently got into dapp development and it's been quite the ride. i am looking forward to collaborating with you all and can't wait to see what we could acheive together!

**Reply by phipsae** [reply]
Welcome!!! :)

**Comment by gboigwe** [comment]
Greetings to everyone - Batch 11 Mentors and my fellow Builders

My name is Gbolahan, I have been around the system in knowledge but I just got started to actually practicalizing anything, it's been hard but I am pushing through. I trust that this is going to be another breakthrough for me as I bounce in. I hope to a collaborative work with the team, see you at the finish line.

**Reply by phipsae** [reply]
Welcome! It's def a long and though ride, but its worth it! Nice that you are with us :)

**Comment by AbolareRoheemah** [comment]
> 👋 Welcome Batch 11 Builders!
> 
> Philip here, I've been a proud member of BuidlGuidl for the past year now. I am developer, love to learn everything about smart contracts and support the batches program.
> 
> Hope that the batch will be a very fun project for you all to go through. If you have any questions, feel free to ask anytime!

Hello everyone!
I'm Roheemah. A frontend and smart contract developer. I'm honored to be among the great builders in this batch, and I look forward to learning and growing together with the team!

**Reply by phipsae** [reply]
Welcome @AbolareRoheemah! Glad to have you here! 🚀

**Comment by futreall** [comment]
Hi , team ! I'm futreall
I recently participated in SpeedRunEthereum and got to explore Scaffold-ETH and Solidity. I'm interested in everything : smart contracts , dApp development, frontend, and backend.
I want to improve my development skills , learn to collaborate effectively , and build something awesome.
I work from morning till night and truly enjoy the process. Thank you for the opportunity to be part of this team! 🚀

**Reply by phipsae** [reply]
Welcome! Nice to have you here 🎉
Hope you can take some learnings from this program for your future development!

**Comment by Vedant-05** [comment]
Hi everyone 👋 ! I'm Vedant Joshi. I'm excited to join Batch 11 of BuidlGuidl! I'm passionate about blockchain development and looking forward to learning more about smart contracts and Web3 development. I'm eager to collaborate with fellow builders and contribute to the community.
Looking forward to working together and learning from all of you! 🙌

**Reply by phipsae** [reply]
Welcome and happy to have you! :)

**Comment by NJarosz** [comment]
Hey everyone! A little late to the party, but I'm Nick Jarosz.  I did the SpeedRun a while ago, but GitHub is definitely a weak spot of mine, so I'm pretty excited to be a part of the batches and to be working with all of you.

**Reply by phipsae** [reply]
Hey Nick, great to have you on board! I hope the batch program helps you to improve your GitHub skills.
